config/routes.rb

```diff
@@ -77,7 +77,12 @@
 
   resources :im_accounts, only: [:create, :update, :destroy]
 
-  resources :delegates, only: [:index]
+  namespace :admin do
```

Comment:
I think we shouldn't create all kind of admin namespaces here, it is a 'small' application and it would only mess up routing. Later on every admin features like delegates exporting and system wide managing, season settings and maybe group creating should be moved here.